src/shared/utils.ts

```diff
@@ -224,7 +224,7 @@ export function getShortcut(mapping?: string): string {
 
     return mapping.replace(
         "Mod-",
-        /Mac/.test(navigator.userAgent) ? "Cmd-" : "Ctrl-"
+        /Mac|iP(hone|[oa]d)/.test(navigator.platform) ? "Cmd-" : "Ctrl-"
```

Comment:
I cited "ProseMirror" approach in my commit message. [Here it is](https://github.com/ProseMirror/prosemirror-commands/blob/master/src/commands.ts#L643).